src/DependencyManagement/BackgroundDependencySnapshotMaintainer.cs

```diff
@@ -73,7 +73,12 @@ public string InstallAndPurgeSnapshots(Func<PowerShell> pwshFactory, ILogger log
 
                 using (var pwsh = pwshFactory())
                 {
-                    _installer.InstallSnapshot(_dependencyManifest, nextSnapshotPath, pwsh, logger);
+                    _installer.InstallSnapshot(
+                        _dependencyManifest,
+                        nextSnapshotPath,
+                        pwsh,
+                        removeIfEquivalentToLatest: true,
```

Comment:
Could you please add a note why we set this to true? ```removeIfEquivalentToLatest: true```

Reply:
Thank you, added.